src/flyte/_run.py

```diff
@@ -258,6 +259,10 @@ async def _run_remote(self, obj: TaskTemplate[P, R] | LazyEntity, *args: P.args,
             env_kv = run_pb2.Envs(values=kv_pairs)
             annotations = run_pb2.Annotations(values=self._annotations)
             labels = run_pb2.Labels(values=self._labels)
+            raw_data_storage = run_pb2.RawDataStorage(raw_data_prefix=self._raw_data_path)
```

Comment:
i think we shouldn't send these at all if the values aren't defined.  there's no point sending an empty object.